runtime/vm/jvminit.c

```diff
@@ -4066,6 +4067,13 @@ static void closeAllDLLs(J9JavaVM* vm) {
 			entry = (J9VMDllLoadInfo*) pool_nextDo(&aState);
 		}
 	}
+
+	if (NULL != vm->jniCryptoFunctions) {
+		const char * cryptolib_name = "jgskit";
+
+		j9sl_close_shared_library(vm->jniCryptoLibrary);
```

Comment:
Should also free (and NULL) the allocated function table.

Reply:
updated